src/DashboardUI/src/hooks/queries/useApplicationQuery.ts

```diff
@@ -83,3 +88,33 @@ export function useFundingOrganizationQuery(waterRightNativeId: string | undefin
     },
   );
 }
+
+export function useGetApplicationQuery(
+  applicationId: string | undefined,
+  perspective: Parameters<typeof getApplication>[1]['perspective'],
```

Comment:
```suggestion
  perspective: 'applicant' | 'reviewer',
```
Keep it simple maybe? Typescript will let us know if they don't match (if one gets renamed). Much easier to understand than the advanced typescript. I'm not too worried about the little bit of duplication in this case

Reply:
that's fair lol, I'll update this. I had just remembered that this more advanced syntax was possible and wanted to try it out, but it's definitely not necessary here.